app/forms/webauthn_setup_form.rb

```diff
@@ -64,11 +65,14 @@ def consume_parameters(params)
   def name_is_unique
     return unless WebauthnConfiguration.exists?(user_id: @user.id, name: @name)
     if @platform_authenticator
-      errors.add :name, I18n.t('errors.webauthn_platform_setup.unique_name'), type: :unique_name
+      incrementer = 1
+      while WebauthnConfiguration.exists?(user_id: @user.id, name: @name)
+        @name = "#{@name} (#{++incrementer})"
+      end
```

Comment:
@zachmargolis thank you for that! It was starting with (2) when I tested locally, if I remove the + 1 that should start with (1) and go up from there, right?

Reply:
Yup! It's very naive, so for example if a user has "Device", "Device 1", "Device 2" and deletes "Device 1", the next would probably be "Device 2"? it might be fine?